pkg/replica/revision_counter.go

```diff
@@ -91,6 +91,11 @@ func (r *Replica) initRevisionCounter() error {
 }
 
 func (r *Replica) GetRevisionCounter() int64 {
+	// Return 0 since revision counter has been diabled.
```

Comment:
I think this has the minimum changes. Could you explain more about your idea? Thanks.

Reply:
The purpose of thess functions is to CRUD the revision counter. We can consider it as a kind of util function. And it's the caller's responsibility (rather than this CRUD functions) to determine if it needs to use revision counter.
But in general, this is not restrictly required. It's more like a possible suggestion or a discussion. And it won't affect the functionality of this feature.

Reply:
I understand your idea. For example, 'GetRevisionCounter' is exposed as interface, and the caller doesn't know if the revision counter is disabled or enabled. Also these CRUD are associated with the individual replicas. I think in this case, handle it in the CRUD function is better. And the caller doesn't need to worry about this either.

Reply:
After discuss with @yasker, I will revert this part of change and change the caller side to not call CRUD function if the revision counter is disabled. Thanks.